### PR TITLE
Feature/eicnet 826 group about page

### PR DESCRIPTION
**Pull request by lramarojaona**

### CODE REVIEW

**Review by 4hardfire — CHANGES_REQUESTED** [review]
@lramarojaona, because you are injecting services from **oec_group_flex** we need to add the module dependency in **eic_groups.info.yml** otherwise the site install process will fail.

File: lib/modules/eic_groups/src/EICGroupsHelper.php [hunk 1]
```diff
@@ -130,4 +175,104 @@ public function getGroupOperationLinks(GroupInterface $group, $limit_entities =
...
+    $joining_methods = $this->groupFlexGroup->getDefaultJoiningMethods($group);
+    $settings = [];
+    foreach ($joining_methods as $joining_method) {
+      if (isset($joining_method_plugins[$joining_method])) {
+        $settings[] = $joining_method_plugins[$joining_method]->getLabel();
+      }
+    }
+    return $settings;
+  }
+
```
Comment (4hardfire) [inline lib/modules/eic_groups/src/EICGroupsHelper.php:277]: @lramarojaona, what do you think if we create a new helper service for the oec_group_flex module and move these methods?

**Review by 4hardfire — APPROVED** [review]
Seems fine to me now. @lramarojaona could you just do a final check before merging?

**Comment by lramarojaona** [comment]
@4hardfire , as discussed I removed the interface for OECGroupFlexHelper class.

**Pull request merged**